code/game/atoms_movable.dm

```diff
@@ -224,6 +224,12 @@
 		newtonian_move(Dir)
 	if(length(client_mobs_in_contents))
 		update_parallax_contents()
+
+	var/datum/light_source/L
```

Comment:
Tested this. The other call got called first and then this proc got called. So the order stays the same here